pages/refed.tsx

```diff
@@ -17,13 +19,38 @@ function Refed() {
                     </Card.Header>
                     <Card.Body>
                         <p>You were invited to use this application because someone thought it would be interesting and helpful to you.</p>
+                        <p>
+                            After you are logged in with Google you can verify your Minecraft account{' '}
+                            <Tooltip
+                                content={
+                                    <span>
+                                        <HelpIcon />
+                                    </span>
+                                }
+                                type="hover"
+                                tooltipContent={
+                                    <p>
+                                        To connect your Minecraft account, search and click yourself in the search bar. Afterwards click "Your? Claim account."
+                                        to get a full explanation.{' '}
+                                    </p>
+                                }
+                            />{' '}
+                            to get 1 hour of <Link href="/premium">premium</Link> for free to test our service including our{' '}
+                            <Link href="/flipper">auction flipper</Link>.
+                        </p>
+                        <p>
+                            We also provide a mod to use the flipper ingame. You can download it in the <b>#mod-releases</b> channel on our{' '}
+                            <a href="https://discord.gg/wvKXfTgCfb">
+                                <span style={{ color: '#7289da' }}>Discord</span>
+                            </a>
+                            . For help check out the <b>#faq</b> channel or ask in <b>#support</b>.
```

Comment:
the faq is also on /flipper it might be better to link there since some people don't use discord